### PR TITLE
GBFSimpleCharacter now implements MovementMode Tags

### PR DESCRIPTION
**Pull request by jagged-teeth**

### CODE REVIEW

File: Source/GameBaseFramework/Private/Characters/GBFSimpleCharacterWithAbilities.cpp [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 #include "GAS/Abilities/GBFAbilitySet.h"
 #include "GAS/Components/GBFAbilitySystemComponent.h"
+#include "GBFTags.h"
+#include "GameFramework/CharacterMovementComponent.h"
```
Comment (Frickerson) [inline Source/GameBaseFramework/Private/Characters/GBFSimpleCharacterWithAbilities.cpp]: <>

Comment (jagged-teeth) [reply Source/GameBaseFramework/Private/Characters/GBFSimpleCharacterWithAbilities.cpp]: fixed

Comment (Frickerson) [reply Source/GameBaseFramework/Private/Characters/GBFSimpleCharacterWithAbilities.cpp]: GBFTags should not have <>, but just keep ""

Comment (jagged-teeth) [reply Source/GameBaseFramework/Private/Characters/GBFSimpleCharacterWithAbilities.cpp]: image
not modified

Comment (jagged-teeth) [reply Source/GameBaseFramework/Private/Characters/GBFSimpleCharacterWithAbilities.cpp]: done.

File: Source/GameBaseFramework/Private/Characters/GBFCharacter.cpp [hunk 2]
```diff
@@ -3,12 +3,12 @@
 #include "Characters/Components/GBFHealthComponent.h"
 #include "Characters/Components/GBFPawnExtensionComponent.h"
 #include "GAS/Components/GBFAbilitySystemComponent.h"
-#include "GBFTags.h"
 #include "GameFramework/GBFPlayerState.h"
 
 #include <AbilitySystemGlobals.h>
 #include <Components/CapsuleComponent.h>
 #include <Engine/World.h>
+#include <GBFTags.h>
```
Comment (Frickerson) [inline Source/GameBaseFramework/Private/Characters/GBFCharacter.cpp]: ""

**Review by Frickerson — CHANGES_REQUESTED** [review]

File: Source/GameBaseFramework/Private/Characters/GBFCharacter.cpp (hunk 2 above)
Comment (jagged-teeth) [reply Source/GameBaseFramework/Private/Characters/GBFCharacter.cpp]: yeah just saw that lmao

**Pull request merged**